seatunnel-dist/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,32 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM openjdk:8-alpine
+
+ENV DOCKER true
+ENV TZ Asia/Shanghai
+ENV SEATUNNEL_HOME /opt/seatunnel
+
+ADD ./target/apache-seatunnel-incubating-*-SNAPSHOT-bin.tar.gz /opt/
+
+RUN cd /opt && mv apache-seatunnel-incubating-* seatunnel
```

Comment:
Miss `tar -xvf` ?

Reply:
The ‘ADD’command will automatically extract tar packages.
https://docs.docker.com/engine/reference/builder/#add